packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -17,8 +24,21 @@
       rel="noopener noreferrer"
     >HTML table element</a>.
```

Comment:
Use "an" above vs. "a":

... it renders a**n**
HTML table element.

Reference: https://owl.purdue.edu/owl/general_writing/grammar/articles_a_versus_an.html#:~:text=Use%20%22an%22%20before%20a%20slient,%2C%20%22an%22%20is%20used.

Reply:
good catch, thank you! Fixed!